go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/etcd-io/auger
 
-go 1.22.0
+go 1.23
```

Comment:
Can you please also bump the image sha256 in `Dockerfile` so that it points to `golang:1.23-alpine`: https://hub.docker.com/layers/library/golang/1.23-alpine/images/sha256-42326cc9d2d47379ccc9b660f4d14f33ea0ebf973491cf6d55d5ee3ebb74e8d3?context=explore

Reply:
Good catch! Thanks!